Training/2.0 Libraries.md

```diff
@@ -25,14 +26,19 @@ Make a two-dimensional numpy array. First row will be: [1,2,3] and second row wi
 Using numpy functionality find the mean, minimum and max value of your array. Index the maximum value and assign it to a variable named max. Index the minimum value and assign it to a variable named min. 
 
 ```
+arr = np.array([[1,2,3],[4,5,6]])
+max  = np.argmax(arr)
+min = np.argmin(arr)
 
 ```
 
```

Comment:
Having the np.argmax(arr) selects the highest index of the array (in python, this array is indexed from 0 to 5), so it will not select the highest number in your array. 
Proper solution would be:
arr = np.array([[1,2,3],[4,5,6]])
max  = arr.max()
min = arr.min()
You can also calculate the mean without the trimmed ends